packages/core/src/lib/util.ts

```diff
@@ -248,3 +248,18 @@ export async function rmdir(path: string): Promise<void> {
   await Promise.all(entries.map((entry) => rmdir(join(path, entry))));
   await fs.promises.rmdir(path);
 };
+
+/**
+   * Given a web manifest's URL, the function retrns the web manifest.
+   *
+   * @param {URL} webManifestUrl the URL where the webManifest is available.
+   * @returns {Promise<WebManifestJson}
+   */
+export async function getWebManifest(webManifestUrl: URL): Promise<WebManifestJson> {
+  const response = await fetch(webManifestUrl);
+  if (response.status !== 200) {
+    throw new Error(`Failed to download Web Manifest ${webManifestUrl}.` +
+        `Responded with status ${response.status}`);
```

Comment:
nit: there should be a space in the string  between the two sentences - at the moment it will say:

Failed to download Web Manifest https://www.example.com/manifest.json.Responded with status...